.nvmrc

```diff
@@ -0,0 +1 @@
+12.14.1
```

Comment:
Any reason why you're not using the super-duper latest version of Node.js?

Reply:
12.14.1 was probably the super-duper latest version of Node.js at the time of authorship. I'll update it.

Reply:
It occurred to me after the fact that it might be nice to use `12.14.1` for consistency with our other apps. Something to consider.

Reply:
Too late! Now you have to run two versions of Node!